opensrp-reveal/src/main/java/org/smartregister/reveal/interactor/ListTaskInteractor.java

```diff
@@ -405,10 +405,6 @@ public void markStructureAsIneligible(Feature feature, String reasonUnligible) {
             details.put(Constants.Properties.TASK_STATUS, task.getStatus().name());
             details.put(Constants.Properties.LOCATION_ID, feature.id());
             details.put(Constants.Properties.APP_VERSION_NAME, BuildConfig.VERSION_NAME);
-            task.setBusinessStatus(NOT_ELIGIBLE);
-            task.setStatus(Task.TaskStatus.COMPLETED);
-            task.setLastModified(new DateTime());
```

Comment:
This will not work since client processing is not setup for `Family Registration Ineligible` eventType.

Also for client processing to be able to extract the business status, you must a field on the form with `fieldcode :business_status`.  org.smartregister.reveal.sync.RevealClientProcessor#calculateBusinessStatus